Python/Product/Debugger/Debugger/PythonThread.cs

```diff
@@ -33,31 +33,23 @@ class PythonThread {
             _name = "";
         }
 
-        public Task StepIntoAsync(CancellationToken ct) {
-            return _process.SendStepIntoAsync(_identity, ct);
-        }
+        public Task StepIntoAsync(CancellationToken ct) =>
+            _process.SendStepIntoAsync(_identity, ct);
```

Comment:
I reverted the change.

---
In reply to: [108208788](https://github.com/Microsoft/PTVS/pull/2323#discussion_r108208788) [](ancestors = 108208788)